### ISSUE TITLE
error MSB3721: 命令“"D:\CUDA_manage\CUDA11.8\cuda1\bin\nvcc.exe" 返回代码为 1

### ISSUE THREAD

**Issue by DeletBug** [mlc-ai/mlc-llm#2007]
## ❓ General Questions

I configure it in a Windows environment (powershell)
I selected cuda, vulkan, and flashInfer after running the command "python3../make/gen_cmake_config. py"。

![image](https://github.com/mlc-ai/mlc-llm/assets/58731986/72c4b103-866e-4012-a6f5-217c1cfd2986)

After running "cmake -- build. -- parallel $(nproc)", the following problem was encountered

![image](https://github.com/mlc-ai/mlc-llm/assets/58731986/3de63714-bd95-4fcf-8133-a0389311e7f2)

I really don't know where the problem lies, where to start and correct it,This is my first time seeking help on GitHub. If there is anything unclear, please forgive me.Hope to receive an answer.

**Comment by MasterJH5574** [comment]
Thank you @DeletBug. We have deprecated the CUDA 11 support. Likely you will need to use CUDA 12 first.

**Comment by DeletBug** [comment]
> Thank you @DeletBug. We have deprecated the CUDA 11 support. Likely you will need to use CUDA 12 first.

Thank you for your answer. I have switched to cuda12.1 now, but unfortunately, the problem has not been resolved yet, it is still the same problem. I'm not sure if there may be a problem elsewhere, could you provide more empirical answers.

![image](https://github.com/mlc-ai/mlc-llm/assets/58731986/a0d6d4b5-eecf-4891-b3bd-10801a4c286a)

 THANKS!

**Comment by MasterJH5574** [comment]
Okay I'm not sure if it is TVM having the thrust issue with Windows. Could you simply open `build/config.cmake` and remove `set(USE_THRUST ON)` from the file? Then thrust will not be enabled and the erroring file will not be compiled. Let me know if this helps.

**Comment by panghongtao** [comment]
请问你是否可以看中文？关于这个项目我想和你多一些探讨是否可以？QQ 853442784 我的QQ联系方式 希望能和你沟通这个项目

**Comment by DeletBug** [comment]
> 好的，我不确定是 TVM 在 Windows 上有推力问题。您可以简单地打开并从文件中删除吗？然后推力将不被启用，错误文件将不会被编译。如果有帮助，请告诉我。`build/config.cmake``set(USE_THRUST ON)`

Hello, there were two issues after switching to version 12.1. The last time I replied to you, I only took a screenshot of one issue. Overall, the two issues after switching to the CUDA version are the same as the two issues I initially raised. (Here is a supplement to the previous answer that did not retain screenshots)

![image](https://github.com/mlc-ai/mlc-llm/assets/58731986/a3e8264e-7a87-41e5-b6d1-d1bf01f77600)

Based on your suggestion, I set the following two to OFF and successfully generated the "mlc_llm. dll" file
![image](https://github.com/mlc-ai/mlc-llm/assets/58731986/b5969dd2-d2d5-442c-8086-604cdf33cc4a)
 
There should be an issue when starting the Windows thrust (at least in my environment). When running "python3../make/gen_cmake_config. py", the last option for "flash transfer" should be to select "no"

thank you very much.
